de/translation.json

```diff
@@ -304,9 +302,9 @@
   "Common.FireteamMembers": "Einsatztrupp-Mitglieder",
   "Common.FireteamReserves": "Reservierungen",
   "Common.FireteamSize": "Einsatztruppgröße",
-  "Common.Friends": "🦘 Friends",
+  "Common.Friends": "Freunde",
   "Common.GloryPoints": "Ruhmpunkte",
-  "Common.GuardianRank": "🦘 Guardian Rank",
+  "Common.GuardianRank": "Hüter Rang",
```

Comment:
`Common.Guardians` is translated as Hüter, so I assume that matches the in game translation